include/NZSL/Ast/Nodes.hpp

```diff
@@ -117,10 +117,18 @@ namespace nzsl::Ast
 
 	struct NZSL_API CallFunctionExpression : Expression
 	{
+		enum class ParameterSemantic : Nz::UInt8
```

Comment:
I'm not sure this enum should be part of CallFunctionExpression

Reply:
Moved out the enum